src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/profiler/ProfilerPresenter.java

```diff
@@ -233,6 +241,39 @@ public void onSaveProfileAs()
    {
       commands_.saveSourceDocAs().execute();
    }
+   
+   public void buildHtmlPath(final OperationWithInput<ProfileOperationResponse> continuation,
+                             final OperationWithInput<Void> onError,
```

Comment:
Idiomatically we usually use `Operation` rather than `OperationWithInput<Void>`.